ca/server.go

```diff
@@ -202,6 +202,9 @@ func (s *Server) IssueNodeCertificate(ctx context.Context, request *api.IssueNod
 					Membership: api.NodeMembershipAccepted,
 				},
 			}
+			if request.InitialSpec != nil {
+				node.Spec.Availability = request.InitialSpec.Availability
```

Comment:
Place comment here indicating that these section restricts the spec fields used.